index.html

```diff
@@ -59,7 +77,7 @@ <h2>I`m a sofware developer</h2>
 
                 <section class="my-recent-work">
                   <h2>My Recent Works</h2>
```

Comment:
- [ ] Kindly fix my works section of your design so it looks like [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
- Your Design
![Capturets](https://user-images.githubusercontent.com/81444338/172576344-b8951115-9615-4d9b-b5b1-7a6c63f03eb6.PNG)
- Figma Design
![Capturefts](https://user-images.githubusercontent.com/81444338/172576378-a61e89db-5601-44ee-8e4e-719b14f8e74f.PNG)